### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #56: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_examples_app.py. (closes #56)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@hachreak there is a problem with `db destoy` run in `test_examples_app.py`. Should be just "create&drop tables" instead?

**Comment by hachreak** [comment]
@jirikuncar :shipit: ?

File: tests/test_examples_app.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    # open page
+    cmd = 'curl http://localhost:5000/records/1'
+    output = subprocess.check_output(cmd, shell=True).decode('utf-8')
+    assert 'Ellis Jonathan' in output
+    cmd = 'curl http://localhost:5000/records/2'
+    output = subprocess.check_output(cmd, shell=True).decode('utf-8')
+    assert 'Ellis Jonathan' in output
+
+
+def test_perms_example_app(perms_app):
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: `test_example_persapp` (or rename persapp.py)

File: tests/test_examples_app.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    """Test example app."""
```
Comment (jirikuncar) [inline tests/test_examples_app.py]: update docstring

File: tests/test_examples_app.py (hunk 1 above)
Comment (hachreak) [inline tests/test_examples_app.py]: done

File: tests/test_examples_app.py (hunk 2 above)
Comment (hachreak) [inline tests/test_examples_app.py]: done

**Comment by hachreak** [comment]
@jirikuncar :shipit: ?

**Pull request merged**